### PR TITLE
Update the source for Xcode11-beta compatibility

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://github.com/appium/appium/issues/12724

I'm surprised that the count of changes is so low %)

### CODE REVIEW

**Comment by KazuCocoa** [comment]
I am running ruby lib core test suite (Not fully updated, but probably almost is okay as Xcode 11 🎉  But not good for iOS 13..? (another issue)))
: https://gist.github.com/KazuCocoa/1d1a41bb619dfc97ffd97009573ab104

some behaviour differs from Xcode 10.2 x iOS 12.2/1. I'm making sure they are Xcode 11 or iOS 13.0 simulator issue

---

Summary:
- Xcode 11 x iOS 12.1 case
    - **This method worked with Xcode 11 x iOS 13.0 combination**
    - `@driver.find_element :name, '<enter text>'` did not work in https://gist.github.com/KazuCocoa/1d1a41bb619dfc97ffd97009573ab104#gistcomment-2936401 ( Xcode 11 x iOS 12.1 source) while it worked on Xcode 10.2 x iOS 12.1
        - `@driver.find_element :name, 'Normal'` could detect the same element
        - https://gist.github.com/KazuCocoa/1d1a41bb619dfc97ffd97009573ab104#gistcomment-2936408 is the source of Xcode 10.2 x iOS 12.1
- clipboard does not work on iOS 13.0 simulator
- `handleGetWindowSize` by iOS 13 returns differnet value as iOS 12 and the before

https://gist.github.com/KazuCocoa/1d1a41bb619dfc97ffd97009573ab104#ios-13-issue

**Comment by KazuCocoa** [comment]
updated ^

File: PrivateHeaders/XCTest/XCUIElementQuery.h [hunk 1]
```diff
@@ -52,6 +51,11 @@
 - (unsigned long long)_derivedExpressedType;
 - (id)initWithInputQuery:(id)arg1 queryDescription:(id)arg2 filter:(CDUnknownBlockType)arg3;
 
+// Deprecated since Xcode 11.0
+- (XCElementSnapshot *)elementSnapshotForDebugDescription;
```
Comment (KazuCocoa) [inline PrivateHeaders/XCTest/XCUIElementQuery.h:55]: 👍

**Comment by KazuCocoa** [comment]
Updated https://github.com/appium/WebDriverAgent/pull/171#issuecomment-499319929 
The method worked with Xcode 11-beta x iOS 13.0 simulator combination. Xcode 11 x iOS 12.1 simulator didn't... 
So far, we can ignore it as known issue.

**Review by KazuCocoa — APPROVED** [review]
I found some issues maybe on iOS 13 simulator (Not Xcode 11)
For Xcode 11, this change looks good to me. (I checked removed/added headers as well)

Thanks 🎉

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Thanks for checking the stuff. I believe the current problems described above reflect the changes that Apple made to the their Xcode 11 SDK

**Pull request merged**